### PR TITLE
remove inherit from compse dict, so it can be identified as empty

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by MartinBasti** [comment]
I miss description why we need this

File: atomic_reactor/source.py [hunk 1]
```diff
@@ -57,6 +57,8 @@ def __init__(self, build_path):
         self.go = self.data.get('go') or {}
         self.image_build_method = meth = self.data.get('image_build_method')
         self.inherit = self.compose.get('inherit', False) if self.compose else False
+        if self.compose:
+            self.compose.pop('inherit', None)
```
Comment (twaugh) [inline atomic_reactor/source.py]: I'd like a comment here explaining why this is useful, i.e. it means that self.compose can be evaluated as a bool in order to decide whether any ODCS composes will be created.

Comment (rcerven) [reply atomic_reactor/source.py]: ok will add

**Comment by rcerven** [comment]
no updating RN since it is fix for unreleased feature

**Pull request merged**